[INFO] {'train': {'log_interval': 1000, 'eval_interval': 1000, 'seed': 1234, 'epochs': 20000, 'learning_rate': 0.0002, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 2, 'fp16_run': True, 'lr_decay': 0.999875, 'segment_size': 8192, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0}, 'data': {'training_files': '/home/ubuntu/alpaco/anichat/tts/vits/filelists/conan_audio_text_train_filelist_ms.txt.cleaned', 'validation_files': '/home/ubuntu/alpaco/anichat/tts/vits/filelists/conan_audio_text_val_filelist_ms.txt.cleaned', 'text_cleaners': ['korean_cleaners'], 'max_wav_value': 32768.0, 'sampling_rate': 22050, 'filter_length': 1024, 'hop_length': 256, 'win_length': 1024, 'n_mel_channels': 80, 'mel_fmin': 0.0, 'mel_fmax': None, 'add_blank': True, 'n_speakers': 3, 'cleaned_text': True}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0.1, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [8, 8, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'n_layers_q': 3, 'use_spectral_norm': False, 'gin_channels': 256}, 'model_dir': './logs/conan_ms'}
[WARNING] /home/ubuntu/alpaco/anichat/tts/vits is not a git repository, therefore hash value comparison will be ignored.
[INFO] Loaded checkpoint './logs/conan_ms/G_457000.pth' (iteration 817)
[INFO] Loaded checkpoint './logs/conan_ms/D_457000.pth' (iteration 817)
/home/ubuntu/anaconda3/envs/alpaco/lib/python3.8/site-packages/torch/functional.py:606: UserWarning: stft will soon require the return_complex parameter be given for real inputs, and will further require that return_complex=True in a future PyTorch release. (Triggered internally at  /opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/native/SpectralOps.cpp:800.)
  return _VF.stft(input, n_fft, hop_length, win_length, window,  # type: ignore[attr-defined]
/home/ubuntu/anaconda3/envs/alpaco/lib/python3.8/site-packages/torch/functional.py:606: UserWarning: ComplexHalf support is experimental and many operators don't support it yet. (Triggered internally at  /opt/conda/conda-bld/pytorch_1659484683044/work/aten/src/ATen/EmptyTensor.cpp:31.)
  return _VF.stft(input, n_fft, hop_length, win_length, window,  # type: ignore[attr-defined]
/home/ubuntu/anaconda3/envs/alpaco/lib/python3.8/site-packages/torch/autograd/__init__.py:173: UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [1, 9, 96], strides() = [3936, 96, 1]
bucket_view.sizes() = [1, 9, 96], strides() = [864, 96, 1] (Triggered internally at  /opt/conda/conda-bld/pytorch_1659484683044/work/torch/csrc/distributed/c10d/reducer.cpp:312.)
  Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
[INFO] Train Epoch: 817 [14%]
[INFO] [1.789027214050293, 3.6620984077453613, 8.949259757995605, 25.280597686767578, 1.4906716346740723, 1.7881042957305908, 910000, 0.00018037912987560822]
[INFO] Saving model and optimizer state at iteration 817 to ./logs/conan_ms/G_910000.pth
[INFO] Saving model and optimizer state at iteration 817 to ./logs/conan_ms/D_910000.pth
[INFO] ====> Epoch: 817
[INFO] Train Epoch: 818 [4%]
[INFO] [2.019832134246826, 3.6043052673339844, 13.538087844848633, 26.911081314086914, 1.5560128688812256, 2.373575448989868, 911000, 0.00018035658248437376]
[INFO] Saving model and optimizer state at iteration 818 to ./logs/conan_ms/G_911000.pth
[INFO] Saving model and optimizer state at iteration 818 to ./logs/conan_ms/D_911000.pth
./logs/conan_ms/G_457000.pth
./logs/conan_ms/D_457000.pth
Traceback (most recent call last):
  File "train_ms.py", line 295, in <module>
    main()
  File "train_ms.py", line 51, in main
    mp.spawn(run, nprocs=n_gpus, args=(n_gpus, hps,))
  File "/home/ubuntu/anaconda3/envs/alpaco/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 240, in spawn
    return start_processes(fn, args, nprocs, join, daemon, start_method='spawn')
  File "/home/ubuntu/anaconda3/envs/alpaco/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 198, in start_processes
    while not context.join():
  File "/home/ubuntu/anaconda3/envs/alpaco/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 160, in join
    raise ProcessRaisedException(msg, error_index, failed_process.pid)
torch.multiprocessing.spawn.ProcessRaisedException: 

-- Process 0 terminated with the following error:
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/alpaco/lib/python3.8/site-packages/torch/multiprocessing/spawn.py", line 69, in _wrap
    fn(i, *args)
  File "/home/ubuntu/alpaco/anichat/tts/vits/train_ms.py", line 119, in run
    train_and_evaluate(rank, epoch, hps, [net_g, net_d], [optim_g, optim_d], [scheduler_g, scheduler_d], scaler, [train_loader, eval_loader], logger, [writer, writer_eval])
  File "/home/ubuntu/alpaco/anichat/tts/vits/train_ms.py", line 189, in train_and_evaluate
    loss_fm = feature_loss(fmap_r, fmap_g)
  File "/home/ubuntu/alpaco/anichat/tts/vits/losses.py", line 13, in feature_loss
    loss += torch.mean(torch.abs(rl - gl))
RuntimeError: The size of tensor a (5072) must match the size of tensor b (4864) at non-singleton dimension 2

